examples/digits_dann_lightn/main.py

```diff
@@ -46,7 +46,11 @@ def main():
         DigitDataset(cfg.DATASET.SOURCE.upper()), DigitDataset(cfg.DATASET.TARGET.upper()), cfg.DATASET.ROOT
     )
     dataset = MultiDomainDatasets(
-        source, target, config_weight_type=cfg.DATASET.WEIGHT_TYPE, config_size_type=cfg.DATASET.SIZE_TYPE
+        source,
+        target,
+        config_weight_type=cfg.DATASET.WEIGHT_TYPE,
+        config_size_type=cfg.DATASET.SIZE_TYPE,
+        val_split_ratio=0.5,
```

Comment:
Better to change it via yaml config rather than hardcode, e.g., cfg.DATASET.VAL_RATIO.

Reply:
To get it running first, you can then config to cfg.DATASET.VAL_RATIO=0.9 in the fast version, but still keeping the default 0.1 as the default in config.py.

Reply:
Sure. I was confused by it having a different name.

Reply:
Would `DATASET.NUM_REPEAT` be another target to reduce?

Reply:
> Sure. I was confused by it having a different name.

What does your "it" refer to? VAL_SPLIT_RATIO? I was hoping to make it more compact but if that causes confusion, just use cfg.DATASET.VAL_SPLIT_RATIO

Reply:
> Would `DATASET.NUM_REPEAT` be another target to reduce?

Oh, I forgot mention that. That's right and we are almost there!
That should be the first to reduce. Just change it to **1** and you will get **10x** improvement. No need to repeat for notebook. Then the notebook should only take minutes to run for 10 classes, and maybe 1 min for 2 classes.

If you think flexible subset is good, we can also make the number of classes in subset a configurable variable.